misc/python/materialize/mzcompose/services/materialized.py

```diff
@@ -141,6 +141,9 @@ def __init__(
                 "--storage-stash-url=postgres://root@cockroach:26257?options=--search_path=storage",
                 "--persist-consensus-url=postgres://root@cockroach:26257?options=--search_path=consensus",
             ]
+            environment += [
+                "MZ_TIMESTAMP_ORACLE_URL=postgres://root@cockroach:26257?options=--search_path=timestamp_oracle"
```

Comment:
ooc, why is this an env var instead of cli arg? seems like it should be the latter if possible

Reply:
It mostly is, but test infra runs older versions of materialize against newer ones to find regressions. And I think this part of test infra can't conditionally pick cli args. The old version would choke on the unknown cli arg, see https://github.com/MaterializeInc/materialize/pull/22262/commits/f457a5bfd2bdeb0bddc9c1c25a7e7abf2f4031b2 for some context.

Reply:
ah, gotcha!